paf-mvp-frontend/src/lib/paf-lib.ts

```diff
@@ -292,8 +303,8 @@ export const refreshIdsAndPreferences = async ({
         thirdPartyCookiesSupported = true;
 
         logger.info('Save "not participating"');
-        setCookie(Cookies.identifiers, PafStatus.NOT_PARTICIPATING, getPrebidDataCacheExpiration());
-        setCookie(Cookies.preferences, PafStatus.NOT_PARTICIPATING, getPrebidDataCacheExpiration());
+        saveCookieValue(Cookies.identifiers, undefined);
+        saveCookieValue(Cookies.preferences, undefined);
 
```

Comment:
shouldn't we set the `lastRefresh` cookie as well?
Otherwise you'll try to refresh everytime you launch the page, don't you?

Reply:
Actually I refresh `lastRefresh` in saveCookieValue function